apps/edge-api/cmd/server/audio_voices_auth_test.go

```
@@ -0,0 +1,206 @@
+			jwtInvoked = true
+			w.WriteHeader(http.StatusUnauthorized)
+		})
+	}
+	next := http.HandlerFunc(func(w http.ResponseWriter, _ *http.Request) {
+		reachedMux = true
+		w.WriteHeader(http.StatusOK)
+	})
+
+	handler := authSelectorMiddleware(jwtMW, next)
+
+	req := httptest.NewRequest(http.MethodGet, audioVoicesPath, nil)
```

golangci-lint (2.13.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)